README.md

```diff
@@ -137,45 +137,47 @@ const result = await multiUserSession.prompt([
 
 Because of their special behavior of being preserved on context window overflow, system prompts cannot be provided this way.
 
-### Emulating tool use or function-calling via assistant-role prompts
+### Function Calling
 
-A special case of the above is using the assistant role to emulate tool use or function-calling, by marking a response as coming from the assistant side of the conversation:
+The Prompt API supports structured function calling using the `tools` option, which allows you to define functions that the language model can call in a language model agnostic way. Each function is described using the `LanguageModelTool` dictionary, which includes an `execute` field specifying the JavaScript function to be called. When the language model requests a function call, the user agent will invoke the specified `execute` function and send the result back to the model.
```

Comment:
Dictionary names are not web developer-facing, so just say something like "an object which includes..."